mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputFormat.java

```diff
@@ -79,8 +81,9 @@ private static HiveIcebergRecordWriter writer(JobConf jc) {
     OutputFileFactory outputFileFactory =
         new OutputFileFactory(spec, fileFormat, location, io, encryption, taskAttemptID.getTaskID().getId(),
             taskAttemptID.getId(), jc.get(HiveConf.ConfVars.HIVEQUERYID.varname) + "-" + taskAttemptID.getJobID());
+    String targetLocation = jc.get(Catalogs.NAME);
```

Comment:
Isn't the location stored under the key `Catalogs.LOCATION`?

Reply:
Good point. The variable name should be `tableName`

Reply:
Fixed